### DISCUSSION TITLE
Quick creation of new issues in Project does not respect changes of target repository

### DISCUSSION THREAD

**Discussion by sirmarcel** [community/community#53204] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

Hello,

Quick report of unexpected behaviour: If new issues are created using the inline "Add item" field in a project, changes in the target repository (given via the `#` shortcut) are not respected. Instead, the "New Issue" popup will *always* default to the last repository for which an issue was created, rather than changing to reflect a different input in the "Add item" field. I've attached a screenshot to illustrate this point. This is not a critical issue, but rather annoying when mass-creating issues.

![Screen Shot 2023-04-19 at 12 06 56](https://user-images.githubusercontent.com/1615910/233042727-588e633a-af4f-4557-bad7-c4583afa1dce.png)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by sirmarcel** [comment]
Small addition: Similar behaviour also happens when it comes to using the "Add item" button in different columns of a Kanban board: New issues are always placed in the previously used column, even if the "Add item" button in a different column was used.

**Comment by azenMatt** [comment, marked as answer]
This has now been resolved and the repo will be pre-populated for you 🎉

**Reply by sirmarcel** [reply]
lovely -- thank you!
